manifests/sftp/user.pp

```diff
@@ -85,7 +90,7 @@
     default => $home,
   }
 
-  if $manage_home {
+  if $manage_home and $ensure == 'present' {
     file {$user_home:
       ensure => directory,
```

Comment:
This should probably also use the `$ensure_dir` variable, however then one needs to be careful with dependencies...